cmd/mesh/rpc_handler.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/0xProject/0x-mesh/core"
 	"github.com/0xProject/0x-mesh/rpc"
 	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/0xProject/0x-mesh/zeroex/ordervalidate"
```

Comment:
Package names are almost always nouns, not verbs. Can we rename this to `ordervalidator`?